utils.js

```diff
@@ -1,31 +1,44 @@
 import axios from "axios";
+import { useContext } from "react";
+import { UserContext } from "./src/components/UserContext";
 
-const newsApi = axios.create({baseURL: 'https://nc-news-5sxy.onrender.com/api' })
+const newsApi = axios.create({
+  baseURL: "https://nc-news-5sxy.onrender.com/api",
+});
 
 export const getArticles = () => {
-    return newsApi.get ('/articles?limit=999').then((res) => {
-        return res.data.articles
-    })
-}
+  return newsApi.get("/articles?limit=999").then((res) => {
+    return res.data.articles;
+  });
+};
 
 export const getArticle = (articleId) => {
-    return newsApi.get (`/articles/${articleId}`).then((res) => {
-        return res.data.article
-    })
-}
+  return newsApi.get(`/articles/${articleId}`).then((res) => {
+    return res.data.article;
+  });
+};
+
+export const patchVote = ({ voteValue, articleId }) => {
+  const voteBody = {
+    inc_votes: voteValue,
```

Comment:
the vote buttons are both disabled when a user has voted - not great UI as if i've upvoted something by accident I can't down vote it and vice versa - what might be a better way to handle this?